autobuild.xml

```diff
@@ -1451,42 +1459,50 @@
               <key>creds</key>
               <string>github</string>
               <key>hash</key>
-              <string>39f52d0350e130f41c5c758f7cb94e87b962c223</string>
+              <string>a9aa3dad944bf031422a2958c6685832a86eceef</string>
               <key>hash_algorithm</key>
               <string>sha1</string>
               <key>url</key>
-              <string>https://api.github.com/repos/secondlife/llphysicsextensions_source/releases/assets/117009336</string>
+              <string>https://api.github.com/repos/secondlife/llphysicsextensions_source/releases/assets/153023528</string>
             </map>
             <key>name</key>
             <string>linux64</string>
           </map>
+          <key>windows</key>
```

Comment:
We can omit 32bit Windows.

And it's shuffling the nodes again! AAAAAAAAAhhhhhhh

Reply:
yeah, that's very annoying.